### ISSUE TITLE
[Feature Request] Take/Put items from controller interface

### ISSUE THREAD

**Issue by AngleWyrm10** [ldtteam/StorageRacks#7]
Great mod, a storage solution that doesn't demand redstone or quartz to start using it.

But it does have a usability issue that's just a bit behind most other offerings: It requires that I first search in the controller GUI for the item I want, and then exit that interface and go open the rack the item is stored in.

Feature Request
Add an upgrade that allows direct access to/from the storage network

**Comment by AngleWyrm10** [comment]
I think that gathering stuff for a build would be problematic, requiring a lot of opening and closing.

**Comment by ProsperCraft** [comment]
Agreed, would love to be able to just grab it right from the interface.
